<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chorus2Dialog</class>
 <widget class="QDialog" name="Chorus2Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>178</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>เอฟเฟ็กต์เสียงประสาน : Chorus</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_10">
   <property name="spacing">
    <number>12</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="SwitchButton" name="btnSwitch" native="true">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>21</height>
        </size>
       </property>
       <property name="toolTip">
        <string>เปิด / ปิด  เสียงประสาน</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnReset">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>12</number>
     </property>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <property name="spacing">
        <number>12</number>
       </property>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="horizontalSpacing">
          <number>2</number>
         </property>
         <item row="0" column="0">
          <widget class="QDial" name="dialDry">
           <property name="minimumSize">
            <size>
             <width>51</width>
             <height>51</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>51</width>
             <height>51</height>
            </size>
           </property>
           <property name="minimum">
            <number>-200</number>
           </property>
           <property name="maximum">
            <number>200</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="notchTarget">
            <double>2.700000000000000</double>
           </property>
           <property name="notchesVisible">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>DryMix</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="spinDry">
             <property name="minimumSize">
              <size>
               <width>61</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>61</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="minimum">
              <double>-2.000000000000000</double>
             </property>
             <property name="maximum">
              <double>2.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_4">
         <property name="horizontalSpacing">
          <number>2</number>
         </property>
         <item row="0" column="0">
          <widget class="QDial" name="dialMin">
           <property name="minimumSize">
            <size>
             <width>51</width>
             <height>51</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>51</width>
             <height>51</height>
            </size>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>6000</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="notchTarget">
            <double>2.700000000000000</double>
           </property>
           <property name="notchesVisible">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>MinSweep (ms)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="spinMin">
             <property name="minimumSize">
              <size>
               <width>61</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>61</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="decimals">
              <number>0</number>
             </property>
             <property name="maximum">
              <double>6000.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>10.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <property name="spacing">
        <number>12</number>
       </property>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <property name="horizontalSpacing">
          <number>2</number>
         </property>
         <item row="0" column="0">
          <widget class="QDial" name="dialWet">
           <property name="minimumSize">
            <size>
             <width>51</width>
             <height>51</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>51</width>
             <height>51</height>
            </size>
           </property>
           <property name="minimum">
            <number>-200</number>
           </property>
           <property name="maximum">
            <number>200</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="notchTarget">
            <double>2.700000000000000</double>
           </property>
           <property name="notchesVisible">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>WetMix</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="spinWet">
             <property name="minimumSize">
              <size>
               <width>61</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>61</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="minimum">
              <double>-2.000000000000000</double>
             </property>
             <property name="maximum">
              <double>2.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_5">
         <property name="horizontalSpacing">
          <number>2</number>
         </property>
         <item row="0" column="0">
          <widget class="QDial" name="dialMax">
           <property name="minimumSize">
            <size>
             <width>51</width>
             <height>51</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>51</width>
             <height>51</height>
            </size>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>6000</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="notchTarget">
            <double>2.700000000000000</double>
           </property>
           <property name="notchesVisible">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>MaxSweep (ms)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="spinMax">
             <property name="minimumSize">
              <size>
               <width>61</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>61</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="decimals">
              <number>0</number>
             </property>
             <property name="maximum">
              <double>6000.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>10.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="0" column="2">
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <property name="spacing">
        <number>12</number>
       </property>
       <item>
        <layout class="QFormLayout" name="formLayout_3">
         <property name="horizontalSpacing">
          <number>2</number>
         </property>
         <item row="0" column="0">
          <widget class="QDial" name="dialFeedback">
           <property name="minimumSize">
            <size>
             <width>51</width>
             <height>51</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>51</width>
             <height>51</height>
            </size>
           </property>
           <property name="minimum">
            <number>-100</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="notchTarget">
            <double>2.700000000000000</double>
           </property>
           <property name="notchesVisible">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Feedback</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="spinFeedback">
             <property name="minimumSize">
              <size>
               <width>61</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>61</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="minimum">
              <double>-1.000000000000000</double>
             </property>
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_6">
         <property name="horizontalSpacing">
          <number>2</number>
         </property>
         <item row="0" column="0">
          <widget class="QDial" name="dialRate">
           <property name="minimumSize">
            <size>
             <width>51</width>
             <height>51</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>51</width>
             <height>51</height>
            </size>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="notchTarget">
            <double>2.700000000000000</double>
           </property>
           <property name="notchesVisible">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Rate (ms/s)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="spinRate">
             <property name="minimumSize">
              <size>
               <width>61</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>61</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="decimals">
              <number>0</number>
             </property>
             <property name="maximum">
              <double>1000.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>10.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SwitchButton</class>
   <extends>QWidget</extends>
   <header>SwitchButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
